pallets/grants/src/tests.rs

```diff
@@ -23,11 +23,20 @@
 use super::*;
 use frame_support::{assert_err, assert_noop, assert_ok, traits::WithdrawReasons};
 use mock::{
-	CancelOrigin, Event as TestEvent, ExtBuilder, Origin, PalletBalances, System, Test as Runtime, Vesting, ALICE, BOB,
+	context_events, CancelOrigin, Event as TestEvent, ExtBuilder, Origin, PalletBalances, System, Test as Runtime,
+	Vesting, ALICE, BOB,
 };
 use pallet_balances::{BalanceLock, Reasons};
 use sp_runtime::DispatchError::BadOrigin;
 
+#[test]
+fn check_releases_default_config() {
+	ExtBuilder::default().build().execute_with(|| {
+		let releases = Releases::default();
+		assert_eq!(releases, Releases::V0);
+		assert_ne!(releases, Releases::V1);
```

Comment:
This test didn't need to be called inside ExtBuilder...execute_with, but not an issue.